src/explore-education-statistics-admin/src/services/featuredTableService.ts

```diff
@@ -0,0 +1,60 @@
+import client from '@admin/services/utils/service';
+
+export interface FeaturedTable {
+  id: string;
+  name: string;
+  description: string;
+  dataBlockId: string;
+  order: number;
+}
+
+export interface FeaturedTableCreateRequest {
+  name: string;
+  description: string;
+  dataBlockId: string;
+}
+
+export interface FeaturedTableUpdateRequest {
+  name: string;
+  description: string;
+}
+
+const featuredTableService = {
+  get(releaseId: string, dataBlockId: string): Promise<FeaturedTable> {
```

Comment:
I was following the pattern we use on the backend - I believe the argument for more conciseness is that you can already see what you're getting because in the code it'll appear as `featuredTableService.get()` - but I also prefer more verbose names.

Would be good if this was consistent across frontend and backend, but probably too late for that now.

Reply:
imo frontend and backend style doesn't/shouldn't need to share the same styling